applications/common/frontend/src/global.scss

```diff
@@ -75,3 +76,11 @@ input[type=checkbox] {
 strong {
   color: inherit;
 }
+
+.is-extra-small {
```

Comment:
I think we already are using `.is-xs-small` class name for content, it just is not in our dropdown.
let's unify the naming